hw/xfree86/parser/patterns.c

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
can you add an SPDX-Licence-Identifier ?

Actually, I'm not a fan of all these huge license text ontop of each file.
And we already have lots of different licenses mixed all over the code ... can we agree on some standard license and just link it in the individual files ?

Reply:
great :)
can you squash this into the original commit that's introducing pattern.c ?